build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -1195,9 +1195,7 @@ CONFIG_ELF_STACKSIZE=2048
 CONFIG_ELF_BUFFERSIZE=32
 CONFIG_ELF_BUFFERINCR=32
 CONFIG_ELF_EXCLUDE_SYMBOLS=y
-CONFIG_ELF_CACHE_READ=y
-CONFIG_ELF_CACHE_BLOCK_SIZE=2048
-CONFIG_ELF_CACHE_BLOCKS_COUNT=60
+# CONFIG_ELF_CACHE_READ is not set
```

Comment:
We should keep this enabled. If this is a part of the commit reverted, then let's add a new commit to keep this enabled.

Reply:
Done.